src/coreclr/gc/gc.cpp

```diff
@@ -6067,7 +6067,7 @@ void gc_heap::release_segment (heap_segment* sg)
     FIRE_EVENT(GCFreeSegment_V1, heap_segment_mem(sg));
     size_t reserved_size = (uint8_t*)heap_segment_reserved (sg) - (uint8_t*)sg;
     reduce_committed_bytes (
-        sg, 
```

Comment:
nit - I see there are some white space changes, are those intentional?

Reply:
Unintended, should not be there.